creation/lib/cvWCreate.py

```diff
@@ -210,7 +288,20 @@ def filter_unwanted_config_attrs(attrs):
             if len(attrs[i].split("=")) > 0:
                 attr = ((attrs[i].split("="))[0]).strip()
             if attr == uattr:
-                attrs[i] = "#%s" % attrs[i]
+                if attr in mlcd:
+                    attrs[i] = "#mlcd  %s" % attrs[i]
+                    suspects = mlcd[attr].split("\n")
+                    for ctr in range(len(suspects)):
+                        for ctr2 in range(len(suspects)):
+                            if attrs[i + ctr + 1].strip() == suspects[ctr2].strip():
+                                attrs[i + ctr + 1] = "#mlcd  %s" % attrs[i + ctr + 1]
+                                break
```

Comment:
Not sure what is happening here. You are changing the values ahead in a for loop?  The loop is not overwriting the value when processing attrs[i+1]?  Probably not because it is not in uattr. It just goes over the same lines but is harmless.
Is the multivalue dumped as multiple lines?  You comment it here all the time not to interfere with the multiline dictionary, correct?
Shouldn't be removed from the multiline dictionary if it is undesired? I.e. should the attributes above be checked in the multiline dictionary keys?

Reply:
refactored and commented so intent is clearer